chains/v2/chains_dev.json

```diff
@@ -1431,6 +1457,34 @@
                     "existentialDeposit": "100000000000",
                     "transfersEnabled": false
                 }
+            },
+            {
+                "assetId": 3,
```

Comment:
id=2 is skipped

Reply:
it is here:
<img width="867" alt="Screenshot 2022-02-03 at 15 40 23" src="https://user-images.githubusercontent.com/40560660/152344620-cbda88ea-0f3b-45d4-9b8d-827cce32acdb.png">
 aUSD has id=2